compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -29,6 +29,8 @@ services:
       - geth
     command: --config-file=/config/beacon.yaml
     ports:
+      - 3500:3500/tcp # for web-ui    
```

Comment:
please add `127.0.0.1` to bind on on localhost and make it inacessible from outside

Reply:
okey

Reply:
`127.0.0.1` is added